### ISSUE TITLE
Use Updated Deeplink Route

### ISSUE THREAD

**Issue by stephancasas** [jonroig/MailtOWA#11]
The new Fluent UI 9 React OWA uses the following route for external `mailto` requests:

```js
`${BASE_URI}/mail/deeplink/compose?mailtouri=${MAILTO_URI}`
```
Where `$BASE_URI` is usually `https://outlook.office365.com` and `$MAILTO_URI` the full `mailto:` URI.

The existing route works, but you may wish to update in case it becomes deprecated.

I pulled this by stepping through the OWA source via debugger.

**Comment by bgpearman** [comment]
Hi @stephancasas, 

Are you able to confirm that your URL schema still works? My testing seems to indicate that it doesn't. For instance, none of the following are successful at drafting emails in outlook web:
 - https://outlook.live.com/mail/action/compose?mailtouri=name@example.com
 - https://outlook.live.com/mail/0/deeplink/compose?mailtouri=name@example.com
 - https://outlook.live.com/mail/deeplink/compose?mailtouri=name@example.com

**Comment by stephancasas** [comment]
Yes, this deeplink still works, but the newer base URL is `https://outlook.office365.com` — not `https://outlook.live.com`.

The full `mailto` URI must be given as well. For example, to send me an e-mail:

```
https://outlook.office365.com/mail/deeplink/compose/?mailtouri=mailto%3astephancasas%40icloud.com
```

Cheers

**Comment by bgpearman** [comment]
Bizzare. All the schemas I have tried either get stuck in a looping gif of the loading mailbox (attached), or if a draft email does open, none of the fields (body/to/...) are filled in. 
![image](https://github.com/jonroig/MailtOWA/assets/64934578/251c4e19-1b95-430b-9796-bda0562d8791)

Your link has the former behaviour for me. When I look at the debugging console, I see the following from the minified JS:
```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'options')
```
